exercises/strings/countLetter/countLetter.js

```diff
@@ -21,13 +21,35 @@
  * @returns {string} The number of times target letter appears in input string
  */
 
+
+// Arian's thoughts://
+ /** 
+  * Ok i remember that to look through a string we need a for loop
+  * Dont remember how that is made so look it up
+  * Great, now that i have looked through the array, how do i count up what the param i'm giving it? 
+  * I think you can just make it a variable and if that variable is equal to it do a simple add fucn
+  * I'm having a hard time conceptualize where to add the letter into the function 
```

Comment:
Imagine the string as a stack of face-down index cards that you're turning over one at a time, with one letter on each index card. You have a post-it note with the letter you're looking for on it labeled `letter`.

You have another post-it note labeled `totalCount` which you'll use to keep track of how many times you see the letter on the post-it note labeled `letter`.

You start by writing `0` on the `totalCount` post-it note.

You flip over the index cards one at a time. For each index card, you look at the letter on that index card. If it matches the letter on the post-it note labeled `letter`, you increment the number on the `totalCount` post-it note by 1. If it doesn't match, you do nothing.

You continue flipping over cards one at a time until you've looked at every character.

When you're done, the post-it note labeled `totalCount` will have the total number of times `letter` appeared in `string` written on it. You hand that post-it note back to whoever asked you to find the answer.